py/bitbox02/communication/generated/btc_pb2.pyi

```diff
@@ -1,10 +1,15 @@
 # @generated by generate_proto_mypy_stubs.py.  Do not edit!
 import sys
+from .common_pb2 import (
+    XPub as common_pb2___XPub,
```

Comment:
Changes in the protobuf generated files without chanigng the proto files?

Reply:
Yeah it's from the previous PR when Multisig was added, but forgot to `make -C py` there.